common/dynamicconfig/constants.go

```diff
@@ -2077,6 +2077,13 @@ the user has not specified an explicit RetryPolicy`,
 		retrypolicy.DefaultDefaultRetrySettings,
 		`DefaultWorkflowRetryPolicy represents the out-of-box retry policy for unset fields
 where the user has set an explicit RetryPolicy, but not specified all the fields`,
+	)
+	EnableReusePolicyRejectOnConflictPolicyTerminate = NewNamespaceTypedSetting(
```

Comment:
Rename to `AllowReusePolicyRejectWithConflictPolicyTerminate`.

Reply:
Well, by using a single flag, this is becoming awkward in terms of wording it.

We want to _deny_ the use of the `RejectDuplicate` together with `TerminateExisting`, but _honor_ the `RejectDuplicateIfFailed` policy.

"Allow" doesn't quite work here because while will "allow" using `RejectDuplicateIfFailed` with `TerminateExisting` - we won't "allow"`RejectDuplicate`.

Open to suggestions.

Reply:
Maybe "Follow..."

Reply:
My comment was just to be the same as your dynamic config name.

Reply:
Ah, that wasn't clear.